parser/src/lib.rs

```diff
@@ -67,6 +67,29 @@ pub fn parse_module<'a, T: FieldElement>(
         .map_err(|err| handle_parse_error(err, file_name, input))
 }
 
+/// Parse an escaped string - used in the grammar.
+pub fn unescape_string(s: &str) -> String {
```

Comment:
is there no utility for this?

Reply:
No. we could use an external crate for that, though.

Reply:
yea better not